addEvent.php

```diff
@@ -214,66 +214,6 @@
                         </tr>
```

Comment:
Great readability once unused code is moved to the bottom. Helpful to have original code in the working file to use for formatting. Make sure to comment throughout.

Reply:
I will be sure to add comments. Do you think there should be comments explaining each field or would that be redundant?

Reply:
It may be helpful to comment on the checkbox fields or any fields that may be difficult to understand. Otherwise, most fields should be easily understood and comments would be redundant.